README.md

```diff
@@ -1 +1,241 @@
+-->
+
+<!-- <div align="center"> -->
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/> -->
+
+  <!-- <h3><b>Portfolio</b></h3>
+
+</div> -->
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Author](#authors)
+- [🔭 Future Features](#future-features)
+- [📝 License](#license)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Portfolio <a name="about-project"></a>
+
+<!-- > Describe your project in 1 or 2 sentences. -->
+> **This project** is about making Amar Muha's developer portfolio website using html and css
+
+<br>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- > Describe the tech stack and include only the relevant sections that apply to your project. -->
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML AND CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+- **Mobile Responsive**
+<!-- - **[key_feature_3]** -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+<br>
+<!-- > Describe how a new developer could make use of your project.
+To get a local copy up and running, follow these steps. -->
+
+<!-- ### Prerequisites -->
+
+<!-- > In order to run this project you need: -->
+
+<!--
+Example command:
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+> Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/amarmahdi1112/portfolio-amar-muha.git
+```
+<!--
+Example commands:
+--->
+
+<!-- ### Install
+
+Install this project with: -->
+
+<!--
+Example command:
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project:
+
+```sh
+  Go to '[root-directory]/my-folder/portfolio-amar-muha/main/' folder and execute the 'index.html' file
+
+```
+  > You can use the [live](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) server from vscode to run the index file 
+<!--
+Example command:
+--->
+
+<!-- ### Run tests
+
+To run tests, run the following command: -->
+
+<!--
+Example command:
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+<!-- ### Deployment
+
+You can deploy this project using: -->
+
+<!--
+Example:
+```sh
+```
```

Comment:
- [ ] since these sections are not relevant to your project you can remove them rather than leaving commented code in your readme file :+1: